app/_src/using-mesh/managing-ingress-traffic/builtin-k8s.md

```diff
@@ -32,9 +33,34 @@ spec:
   tags:
     kuma.io/service: edge-gateway
 ```
+{% endif_version %}
+{% if_version gte:2.7.x %}
+{% if_version lte:2.9.x %}
+[//]: # (This is change in behavior, let's assume that users will get used to it, so we won't have to show this warning after 2.9.x)
```

Comment:
do we need to make sure to disallow this after after 2.9 so it does not pass validation?

Reply:
There is an issue for that: https://github.com/kumahq/kuma/issues/9635